man/systemd.net-naming-scheme.xml

```diff
@@ -531,6 +531,22 @@
     particular version of systemd).</para>
   </refsect1>
 
+  <refsect1>
+    <title>Limit reading specific sysfs attributes</title>
+
+    <para>When creating names for network cards, some naming schemes use data from sysfs populated
+    by the kernel. This means that although a specific naming scheme in udev is picked,
+    the network card's name can still change when a new kernel version adds a new sysfs attribute.
+    For example if kernel starts setting <constant>phys_port_name</constant>, udev will append the
```

Comment:
maybe say explicitly: "… setting the <constant>phys_port_name</constant> sysfs attribute, udev …"